scripts/migrate_from_givtcp.py

```
@@ -579,6 +952,160 @@ async def migrate_entity(
+    cutover: datetime,
+    ge_unit: str,
+    apply: bool,
+    ge_known: bool,
+) -> MigrationResult:
+    r = MigrationResult(description, ge_id)
+    try:
+        raw_givtcp = await ws.get_statistics(
+            [givtcp_id], _EPOCH, end=cutover, types=["mean", "min", "max"]
+        )
+        raw_ge = await ws.get_statistics([ge_id], _EPOCH, types=["mean", "min", "max"])
+    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)